doc/filters/slug.rst

```diff
@@ -0,0 +1,29 @@
+``slug``
+========
+
+The ``slug`` filter transforms a given string into another string that
+only includes safe ASCII characters. It does so by using the method by
```

Comment:
I would move the second sentence at the end (as most people don't care about how it is implemented).